lcd/screen.h

```diff
@@ -4,7 +4,7 @@
 
 #include <cstdint>
 
-#define KEYBOARD_INPUT_BUFFER_SIZE 3
+#define KEYBOARD_INPUT_BUFFER_SIZE 8
```

Comment:
Why is this 8?  Why was it 3?  Unclear why this is reasonable or why it was chosen